src/deluge/dsp/compressor/rms_feedback.cpp

```diff
@@ -45,7 +45,7 @@ void RMSFeedbackCompressor::renderVolNeutral(StereoSample* buffer, uint16_t numS
 	// this is a bit gross - the compressor can inherently apply volume changes, but in the case of the per clip
 	// compressor that's already been handled by the reverb send, and the logic there is tightly coupled such that
 	// I couldn't extract correct volume levels from it.
-	render(buffer, numSamples, 2 << 26, 2 << 26, finalVolume >> 3);
+	render(buffer, numSamples, ONE_Q31 / 16, ONE_Q31 / 16, finalVolume >> 3);
```

Comment:
I think this and the change to a right shift by 4 below are cancelling eachother out so the overall gain change is 0db. This gain reduction is a bit of a hack, but without it something clips internally and we get a high pitched whine -- maybe the RMS calculation, because that sums potentially 64 q31s?